ci-operator/config/openshift/cluster-ingress-operator/openshift-cluster-ingress-operator-master.yaml

```diff
@@ -143,6 +143,23 @@ tests:
         requests:
           cpu: 100m
     workflow: ipi-aws
+- always_run: false
+  as: e2e-aws-gatewayapi-conformance
+  optional: true
+  steps:
+    cluster_profile: aws
+    env:
+      FEATURE_GATES: '[GatewayAPI=true]'
```

Comment:
You didn't need to add the workaround feature gate `ClusterAPIInstallAWS` here, as you did in https://github.com/openshift/release/pull/52547 for other tests.  Should it be removed from the other tests too?

Reply:
Yes, the feature gate `ClusterAPIInstallAWS` could be removed since it is temp workaround for 4.16, but it is harmless and we could remove it in another PR. It was added for the job `e2e-aws-gatewayapi` and the job could be removed once GWAPI is promoted to TechPreview/GA, I think.